.travis.yml

```diff
@@ -15,17 +15,19 @@ git:
   depth: 10
 
 before_script:
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then chmod +x benches-compare.sh; fi
```

Comment:
I changed the benches to only run on nightly [here](https://github.com/purpleprotocol/graphlib/commit/069610b8b73fc6e083bea9b2ef65b50fd8b0e9d2#diff-354f30a63fb0907d4ad57269548329e3R18). There is no point on running them on stable as it increases the build time considerably.

Reply:
done